android/KMEA/app/build.gradle

```diff
@@ -32,6 +32,11 @@ android {
         }
     }
 
+    // TODO: Remove this when SDK > 17
+    lintOptions {
+        disable 'ResourceType'
```

Comment:
Maybe we want to disable the same lint-checks for KMEA and KMAPro?

```
  lintOptions {
        disable 'MissingTranslation'
    }
```

Reply:
Yes, it would probably be good to have some consistency in the projects on which lint errors we choose to ignore.

OTOH, I'd lean towards minimizing the number of places we're ignoring lint errors. In an ideal world, we fix the code and not have to ignore any of these :)